content/manual/running/hosting.md

```diff
@@ -7,196 +7,491 @@ weight = 2
-  - [Digital Ocean Nginx Config][do nginx config]
-  - [Digital Ocean SSL Cert Setup][do ssl config]
-  - [Urbit Install Docs](https://urbit.org/getting-started/)
-  - [Urbit Basic Cloud Install][urbit basic cloud install]
-
-[gandi]: https://www.gandi.net/
-[digital ocean]: https://www.digitalocean.com/
-[do dns]: https://www.digitalocean.com/community/tutorials/how-to-point-to-digitalocean-nameservers-from-common-domain-registrars
-[do nginx install]: https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04
-[do nginx config]: https://www.digitalocean.com/community/tutorials/how-to-deploy-a-go-web-application-using-nginx-on-ubuntu-18-04
-[do ssl config]: https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-18-04
-[urbit basic cloud install]: https://medium.com/@urbitlive/hello-world-urbit-edition-install-boot-and-run-your-urbit-planet-on-a-10-cloud-server-b9579745b9a8
-[do initial setup]: https://www.digitalocean.com/community/tutorials/initial-server-setup-with-ubuntu-18-04
-[blog github]: https://github.com/zalberico/zalberico.github.io
-
-If you plan to use Nginx as a reverse proxy for your Urbit, it is important that you include the following settings in your configuration in order to allow the Landscape web client to properly communicate with your Urbit:
-` chunked_transfer_encoding off; proxy_buffering off; proxy_cache off; `
+The goal of this guide is to have clear and easy to follow best practices for
+deploying an Urbit node to a server you control in the cloud. Deploying in the
+cloud allows you to access your Urbit from any device.
+
+Most Urbit users start out running their ship locally on one machine in order to
+play with it, but this means when your machine is offline your Urbit node is
+offline too (and can't get updates). You can also only access your Urbit from
+that one machine.
+
+This guide uses Digital Ocean as the cloud provider, but others can be used. If
+using another provider, the setup script provided and other server configuration
+instructions may need to be modified or done manually.
+
+## 1. Create a Droplet
+
+Create an account on [Digital Ocean](https://digitalocean.com). Once you make an
+account, choose "Deploy a virtual machine".
+
+You should see the page below where you can create your Droplet, aka Virtual Machine:
+
+![do screenshot](https://media.urbit.org/operators/manual/running/hosting/do-screenshot.png)
+
+Fill out the options like so:
+
+#### Image
+
+Ubuntu 22.04 x64
+
+#### Plan
+
+- Shared CPU: Basic
+- CPU options: Regular with SSD
+- 2GB / 1 CPU ($12/mo)
+
+You can choose a more powerful option if you'd like but the $12 option should be
+sufficient. Note Urbit needs 2GB of memory; it's possible to choose a cheaper
+option and run it with less memory by using swap but it will impact performance.
+
+#### Add block storage
+
+The $12 plan includes 50GB which should be sufficient for quite some time, so
```

Comment:
May want to include the plan name in case pricing changes in the near term.

Reply:
they don't have plan names for these sub-options unfortunately